sklearn/covariance/empirical_covariance_.py

```diff
@@ -69,7 +69,8 @@ def empirical_covariance(X, assume_centered=False):
     """
     X = np.asarray(X)
     if X.ndim == 1:
-        X = np.reshape(X, (1, -1))
+        X = X[:, np.newaxis]
```

Comment:
On 08/27/2015 09:16 PM, Vighnesh Birodkar wrote:
>
> I thought it was safe to do that because the test cases passed, you 
> are worried of breaking someone's code with this fix, is it ?
>
Yes.